src/main/java/org/cloudbees/literate/jenkins/promotions/PromotionTrigger.java

```diff
@@ -136,7 +148,11 @@ public ListBoxModel doFillProcessItems(@AncestorInPath Item defaultJob,
 
             LiterateMultibranchProject j = null;
             if (jobName != null) {
-                j = Jenkins.getInstance().getItem(jobName, defaultJob, LiterateMultibranchProject.class);
+                Jenkins jenkins = Jenkins.getInstance();
+                if (jenkins == null) {
+                    throw new IllegalStateException(); // TODO 1.590+ getActiveInstance
```

Comment:
please add a message - if a user ever sees this having ("Jenkins is starting up or shutting down") is highly valuable.